docs/usage/1-routing/5-mounting-asgi-apps.md

```diff
@@ -11,7 +11,7 @@ The handler function will receive all requests with an url that begins with `/so
 `/some/sub-path/abc` and `/some/sub-path/123/another/sub-path` etc.
 
 !!! info Technical Details
-    If we were to send a request to the above with the url `/some/sub-path/abc`, the handler will be invoked and
+    If we were to send a request to the above with the url `/some/sub-path`, the handler will be invoked and
```

Comment:
```suggestion
    If we are sending a request to the above with the url `/some/sub-path`, the handler will be invoked and
```
Otherwise the sentence reads a bit weird